sdk-extensions/metric-incubator/src/test/java/io/opentelemetry/sdk/viewconfig/ViewConfigCustomizerTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.viewconfig;
+
+import static io.opentelemetry.sdk.testing.assertj.MetricAssertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.testing.exporter.InMemoryMetricReader;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class ViewConfigCustomizerTest {
+
+  @Test
+  void customizeMeterProvider_Spi() {
+    System.setProperty("otel.traces.exporter", "none");
+    System.setProperty(
+        "otel.experimental.metric.view.config",
+        ViewConfigTest.class.getResource("/view-config-customizer-test.yaml").getFile());
```

Comment:
I think there is huge demand to be able to support bundled resources with this sort of configuration property. It seems we can copy spring's practice, how about we add support for `classpath:`?

https://www.baeldung.com/spring-classpath-file-access#2-using-value

Reply:
Good idea. Do you foresee any issuing accessing resources when this is packaged with the agent?